src/frontend/src/components/public/PIAListTable/index.tsx

```diff
@@ -52,9 +52,16 @@ const PIAListTable = ({ headings, pias, sorting }: IDataTable) => {
           {pias.map((pia, index) => (
             <tr key={index}>
               <td
+                // tabIndex={0}
```

Comment:
Guess we do need tabIndex. Remove comment

Reply:
yes, i add it back